src/status_im/chat/screen.cljs

```diff
@@ -73,25 +73,45 @@
                                :group-chat group-chat
                                :current-public-key current-public-key)])
 
-(defview messages-view [group-chat]
+
+(defview messages-view [chat-id group-chat]
   (letsubs [messages           [:get-current-chat-messages]
-            current-public-key [:get-current-public-key]]
-    [list/flat-list {:data                      messages
-                     :render-fn                 (fn [{:keys [message-id] :as message}]
-                                                  ^{:key message-id} [message-row {:group-chat         group-chat
-                                                                                   :current-public-key current-public-key
-                                                                                   :row                message}])
-                     :inverted                  true
-                     :onEndReached              #(re-frame/dispatch [:load-more-messages])
-                     :enableEmptySections       true
-                     :keyboardShouldPersistTaps (if platform/android? :always :handled)}]))
+            current-public-key [:get-current-public-key]
+            opacity            (anim/create-value 0)
+            duration           (if platform/android? 100 200)
+            timeout            (if platform/android? 50 0)
+            preview-style      {:flex            1
+                                :align-items     :center
+                                :justify-content :center}
+            preview            (vec [react/view preview-style])]
+    {:component-did-mount (fn [component]
+                            (js/setTimeout
+                             #(anim/start (anim/spring opacity {:toValue  1
```

Comment:
Why are we adding animation stuff?

Reply:
This is not required at all, but otherwise when messages-view gets rendered it's blinking a bit, especially this is noticeable on faster devices. This animation smooths out appearance of messages.